src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -8360,6 +8360,363 @@ .maxstack  2
   IL_003a:  blt.s      IL_000e
   IL_003c:  ret
 }
+");
+        }
+
+        [Fact]
```

Comment:
> [Fact] [](start = 7, length = 7)

Are patterns in initializers implemented in a different branch? If they are in this branch we should add similar tests for patterns as well here. See `PatternVariable_*` tests above.